stdlib/stdlib.mli

```diff
@@ -51,6 +51,12 @@ exception Exit
 (** The [Exit] exception is not raised by any library function.  It is
     provided for use in your programs. *)
 
+val try_finally : (unit -> 'a) -> (unit -> unit) -> 'a
+(** [try_finally f g] calls [f ()] and returns its result.  If it raises,
+    the same exception is raised; in {b: any} case, [g ()] is called after
```

Comment:
What you write here does not hold. If `g` raises then it's not the same exception that is raised, it's the exception of `g`.

Reply:
True. I'd rather reword it than hide the exception from `g`

Reply:
> True. I'd rather reword it than hide the exception from g, what do you think?

Yes it's not a good idea, as you could hide e.g. an `Out_of_memory` or asynchronous exceptions like `Sys.break`.

Reply:
Yes, that's exactly what I was thinking about